Sprint-1/destructuring/exercise-1/readme.md

```diff
@@ -30,4 +30,16 @@ console.log(`Batman is ${firstName}, ${lastName}`);
 # Exercise
 
 - What is the syntax to destructure the object `personOne` in exercise.js?
+const personOne = {
+  name: "Popeye",
+  age: 34,
+  favouriteFood: "Spinach",
+};
+const {name,age,favouriteFood}=personone
```

Comment:
Javascript is case sensitive. Please, correct the mistake related to case in the name of your constant. Also, please, apply proper spacing formatting.